exampleSite/content/_global/nav.md

```diff
@@ -5,8 +5,11 @@ date = "2018-05-17"
 weight = 0
 background = "dark"
 search = true
-breadcrumb = true # Default value is false
-#breadcrumb_level = 0 # Default is 1
+
+[breadcrumb]
+  display = true # Default value is false
+  level = 0 # Default is 1
```

Comment:
What does `0` do?

Reply:
I commented it out. It's the level from which we show the crumbs. Since breadcrumbs are displayed in order to show parents, I provided this variable to customize from which parent we should display the crumbs. For example home is always at the level 0, every page in root section is level 1. If those pages are sections containing other pages, those pages are level 2 and so on.

Reply:
Ah I was thinking at first thought it's the depth level.

Reply:
Probably best to add a comment here.